packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -62,15 +62,24 @@ export function isLinux() {
   return os.type() === "Linux";
 }
 
+export interface TeamsAppTelemetryInfo {
+  appId: string;
+  tenantId: string;
+}
+
 // Only used for telemetry when multi-env is enabled
-export function getTeamsAppIdByEnv(env: string) {
+export function getTeamsAppTelemetryInfoByEnv(env: string): TeamsAppTelemetryInfo | undefined {
   try {
     const ws = ext.workspaceUri.fsPath;
 
     if (isValidProject(ws)) {
       const result = environmentManager.getEnvStateFilesPath(env, ws);
       const envJson = JSON.parse(fs.readFileSync(result.envState, "utf8"));
-      return envJson[PluginNames.APPST].teamsAppId;
+      const appstudioState = envJson[PluginNames.APPST];
+      return {
+        appId: appstudioState.teamsAppId,
+        tenantId: appstudioState.tenantId,
```

Comment:
I remember tenant id is not included in app studio, it's in solution instead.

![image](https://user-images.githubusercontent.com/71362691/155091604-40256285-2aa8-4b9b-a4c1-f79b8896567e.png)

Reply:
Just check the latest `dev` bits, the `tenantId` config is also included in AppStudio plugin now.
So we have the same `tenantId` in 3 components' config: solution, AAD and AppStudui, we'd better consolidate this since the value here should be the same.